src/_locales/en/messages.json

```
@@ -263,7 +263,7 @@
   },
 
   "optionCopyEmojiColons": {
-    "message": "Use the $COLON$ code instead of the Emoji unicode.",
+    "message": "Use the $COLON$ shortcode instead of the Emoji Unicode.",
```

You are right, but when we are at already at it, do it properly then: :stuck_out_tongue_winking_eye: 

```suggestion
    "message": "Use the $COLON$ shortcode instead of the Unicode Emoji.",
```

And to adjust the settings page for an optional permission, I already have a JS module for that… :thinking:

---

No problem. Done in https://github.com/rugk/awesome-emoji-picker/pull/93/commits/16600e12b4c27cc34a8a9dd34f68df21ad34e864.